bin/python/ripple/util/Sign.py

```diff
@@ -113,6 +116,16 @@ def get_signature(seq, validator_public_key_human, private_key_human):
     m1 = sign_manifest(m, private_key, pk)
     return base64.b64encode(m1)
 
+def verify_signature(seq, validator_public_key_human, public_key_human, signature):
+    v, validator_public_key = Base58.decode_version(validator_public_key_human)
+    check_validator_public(v, validator_public_key)
+
+    v, public_key = Base58.decode_version(public_key_human)
+
+    m = make_manifest(public_key, validator_public_key, seq)
+    public_key = public_key[1:]     # Remove ED25519_BYTE
+    sig = signature.decode('hex')
+    ed25519.checkvalid(sig, 'MAN\0' + m, public_key)
```

Comment:
Add a `test_verify_signature` function to `test_Sign.py` unit test this function. FYI, to run the python unit tests, go to `bin/python` and run `python -m unittest discover`